NanoAnalysis/python/nanoZZ4lAnalysis.py

```diff
@@ -193,14 +196,19 @@
     insertBefore(reco_sequence, 'lepFiller', getEleScaleRes(LEPTON_SETUP, DATA_TAG, IsMC, overwritePt=True))
 
 # Update of JetId from manual recipe for NanoAOD v12
-if NANOVERSION == 12:
-    from ZZAnalysis.NanoAnalysis.jetIdUpdate import *
-    insertBefore(reco_sequence, 'jetFiller', jetIdUpdate())
+#if NANOVERSION == 12:
```

Comment:
This should still only be done for NANOVERSION>=12 (we still have Run2 data/MC in v9, which would be broken without the if)